iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -44,10 +44,10 @@ final class Toast: NSObject {
   }
 
   @objc func show(in view: UIView?, alignment: Alignment) {
-    guard let view else { return }
+    guard let view = view else { return }
```

Comment:
This is not needed, right? It's more verbose: https://swiftfiddle.com/z2m3gg3wyvc2zmpnf5zayqrafa

Reply:
Needed for the Swift within the XCode 13. I've added a separate commit for that, no problem here with explicit unboxing.